src/Compilers/VisualBasic/Portable/Semantics/AccessCheck.vb

```diff
@@ -103,6 +103,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Case SymbolKind.NamedType
                     Return CheckNamedTypeAccessibility(DirectCast(symbol, NamedTypeSymbol), within, basesBeingResolved, useSiteDiagnostics)
 
+                Case SymbolKind.Alias
+                    Return CheckSymbolAccessibilityCore((DirectCast(symbol, AliasSymbol)).Target, within, Nothing, basesBeingResolved, useSiteDiagnostics)
```

Comment:
>(DirectCast(symbol, AliasSymbol)).Target [](start = 56, length = 40)

Is target always not null?

Reply:
> Is target always not null?

The compilers do not expose alias symbols with a null target.

---
In reply to: [192575133](https://github.com/dotnet/roslyn/pull/27312#discussion_r192575133) [](ancestors = 192575133)

Reply:
> The compilers do not expose alias symbols with a null target.

Are they ever created internally?

---
In reply to: [192885921](https://github.com/dotnet/roslyn/pull/27312#discussion_r192885921) [](ancestors = 192885921,192575133)

Reply:
> Are they ever created internally?

There are circumstances in which the target is computed lazily, but it does not appear a `null` target can be exposed through the `Target` property.